app/src/main/java/com/example/feelgoodinc/fragments/EducationResources.java

```diff
@@ -38,11 +36,11 @@ public PlaceHolderResources() {
      *
      * @param param1 Parameter 1.
      * @param param2 Parameter 2.
-     * @return A new instance of fragment PlaceHolderResources.
+     * @return A new instance of fragment EducationResources.
      */
     // TODO: Rename and change types and number of parameters
-    public static PlaceHolderResources newInstance(String param1, String param2) {
-        PlaceHolderResources fragment = new PlaceHolderResources();
+    public static EducationResources newInstance(String param1, String param2) {
```

Comment:
What are `param1` and `param2` actually for? probably should delete them if they don't do anything